build.gradle.kts

```diff
@@ -34,3 +33,5 @@ allprojects {
 task<Delete>("clean") {
     delete(rootProject.buildDir)
 }
+
+tasks.register("checkGeneratedFiles") {}
```

Comment:
seems it is some leftover?

Reply:
it is actually needed by the shared pipeline (but no use here so it is a task doing nothing)

Reply:
can we then add a comment? so that empty task is not confusing